experimental/Parallel/src/Parallel.jl

```diff
@@ -22,24 +22,38 @@ mutable struct OscarWorkerPool <: AbstractWorkerPool
   wids::Vector{Int}     # a list of the ids of all workers which are associated to this pool
   oscar_channels::Dict{Int, <:RemoteChannel} # channels for sending `type_params` to the workers
 
-  function OscarWorkerPool(n::Int)
-    wids = addprocs(n)
+  function OscarWorkerPool(n::Int; kw...)
+    wids = addprocs(n, kw...)
     wp = WorkerPool(wids)
     # @everywhere can only be used on top-level, so have to do `remotecall_eval` here.
     remotecall_eval(Main, wids, :(using Oscar))
 
     return new(wp, wp.channel, wp.workers, wids, Dict{Int, RemoteChannel}())
   end
+
+  function OscarWorkerPool(manager::ClusterManager; project=nothing, kw...)
```

Comment:
```suggestion
  function OscarWorkerPool(manager::ClusterManager; project=Base.active_project(), kw...)
```
The active project is I think a reasonable default, and matches what happens for local workers.
This way one could still manually pass `nothing` to avoid this step if needed.